Apps/W1/EDocument/app/src/Log/EDocumentIntegrationLogs.Page.al

```diff
@@ -31,7 +31,7 @@ page 6128 "E-Document Integration Logs"
                     {
                         ToolTip = 'Specifies the service code for the document.';
                     }
-                    field(URL; Rec.URL)
+                    field(URL; Rec."Request URL")
```

Comment:
Given we now reference the new field, shouldn't we have upgrade code then? Using DataTransfer that should be a swift procedure.

Reply:
That's true, of course. I had not considered this, as it is "only" a log. However, it is visible to the user, so as a user I would also expect the existing data to be transferred. I will add this.

Reply:
I've added the upgrade code internally. Case closed.